showcase/tests/integration/components/hds/application-state/index-test.js

```diff
@@ -22,5 +26,59 @@ module(
 
       assert.dom('#test-application-state').hasClass('hds-application-state');
     });
+
+    test('it should have the correct alignment class when no alignment is provided', async function (assert) {
+      await render(hbs`
+      <Hds::ApplicationState id="test-application-state">
+        template block text
+      </Hds::ApplicationState>
+    `);
+
+      assert
+        .dom('#test-application-state')
+        .hasClass('hds-application-state--align-left');
+    });
+
+    test('it should have the correct alignment class when alignment is set to "left"', async function (assert) {
+      await render(hbs`
+      <Hds::ApplicationState id="test-application-state" @align="left">
+        template block text
+      </Hds::ApplicationState>
+    `);
+
+      assert
+        .dom('#test-application-state')
+        .hasClass('hds-application-state--align-left');
+    });
+
+    test('it should have the correct alignment class when alignment is set to "center"', async function (assert) {
+      await render(hbs`
+      <Hds::ApplicationState id="test-application-state" @align="center">
+        template block text
+      </Hds::ApplicationState>
+    `);
+
+      assert
+        .dom('#test-application-state')
+        .hasClass('hds-application-state--align-center');
+    });
+
+    test('it should throw an error when alignment is set to an invalid value', async function (assert) {
```

Comment:
Try to use the "standard" way in which we do the assertion tests: https://github.com/hashicorp/design-system/blob/main/showcase/tests/integration/components/hds/alert/index-test.js#L169-L180